app/views/session/_oidc.html.haml

```diff
@@ -1,6 +1,7 @@
+%script{src: "/wakeUp.js"}
 .login-form
   = image_tag "/assets/images/cryptopussy_oidc_sleep.svg", class: 'logo'
   .form-group
     .alert.alert-login-box.alert-info
       = t('.sleep')
-    = link_to(t('.submit'), Rails.application.routes.url_helpers.root_path, class: 'btn btn-primary')
```

Comment:
maybe just leave path if js is somehow disabled? maybe path is not required at all?!

with path, you might need prevent default on submit button.